filebeat/module/apache/access/test/ssl-request.log-expected.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "@timestamp": "2018-08-10T07:45:56.000Z",
+        "@timestamp": "2018-08-10T09:45:56.000Z",
```

Comment:
@jsoriano @spinscale Is this change related to the date changes in Elasticsearch? The original timestamp looks as following: `[10/Aug/2018:09:45:56 +0200]`.

Reply:
Looks like that, yes, #10433 reopened.